exercises/two-fer/two_fer_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'minitest/autorun'
+require_relative 'two_fer'
+
+class TwoFerTest < Minitest::Test
+  def test_empty
```

Comment:
This test file should be the one generated by the test generator you have written.

You can create the file by running: `bin/generate two-fer` from your Exercism ruby directory.

Make sure you have the `problem-specifications` repository checked out.

See: https://github.com/exercism/ruby#generated-test-suites